srml/timestamp/readme.md

```diff
@@ -0,0 +1,81 @@
+# Timestamp Module
+
+The timestamp module provides functionality to get and set the on-chain time.
+
+## Overview
+
+The timestamp module allows the validators to set and validate a timestamp with each block. It uses timestamp data as inherent data which is provided by the block author and validated/verified by other validators.
+
+The timestamp module is the recommended way to query the on-chain time instead of using an approach based on block numbers. The block numbers based time measurement can cause issues because of cummulative calculation errors and hence it should be avoided.
+
+## Public Interface
+
+### Supported origins
+
+* `inherent` - Used to set the timestamp for the current block
+
+### Types
+
+* `Moment` - Represents the current timestamp. Generally implemented as a `u64`.
+
+### Storage Items
+
+* `Now`: `Moment` - The current timestamp. A `u64` having the timestamps as **total seconds from the unix epoch**.
+* `BlockPeriod`: `Moment` - The minimum (and advised) period between blocks. Also represented as `Moment`.
+* `DidUpdate`: `bool` - Private flag to check if the timestamp was updated for the current block.
+
+### Public Inspection functions - Immutable (getters)
```

Comment:
could we make these headings more concise by renaming them to just "Public Getters" and "Public Setters"

Reply:
it looks like they've changed to `Public Immutable functions` and `Public Mutable functions`, which differs from the example template https://github.com/paritytech/substrate/pull/1947/files

Reply:
I found it better this way, will update the template instead.